### ISSUE TITLE
Archive Meyer TissueSensitivity dataset

### ISSUE THREAD

**Issue by mshadbolt** [ebi-ait/hca-ebi-wrangler-central#20]
This is a ticket to track progress towards full archiving of this dataset.

The data was partially archived under:
BioStudies: S-SUBS3  (linked to many biosamples accessions)
ENA: PRJEB31843

We have been attempting to update and add data to it via DSP but keep running into issues.

Once it is fully archived we will need to add the accessions to the existing metadata.

Relevant DSP submission ids:
* a0b21625-e268-47d1-927a-66f2fe2257a9 - submission that contains large fastq files that are failing to validate (~26 sets from WGS data)
* 79e536a6-72bc-4d54-b2be-a4e6a30966f2 -  submission that contains valid data and metadata that should be making its way to archives 

See this issue for previous work https://github.com/HumanCellAtlas/hca-data-wrangling/issues/427

**Comment by ESapenaVentura** [comment]
The dataset has been archived - We are in the process of deciding if we need metadata updates on the ENA library layout.

Accessions are:
[S-SUBS3](https://www.ebi.ac.uk/biostudies/studies/S-SUBS3) : BioStudies - linked to all BioSamples accessions.
[PRJEB31843](https://www.ebi.ac.uk/ena/data/view/PRJEB31843): ENA Study - Contains all data

Both haven't changed from the previous iteration, but contains the whole project now.

Should we close the ticket?

**Comment by lauraclarke** [comment]
This ticket is being reopened as we need to ensure EMBL-EBI archive accessions are added back to the ingest records for this project so they can eventually be updated in the DCP

**Comment by mshadbolt** [comment]
so to be clear, you expect us to update 100s of entities by hand in the UI?

**Comment by clairerye** [comment]
I am guessing I have misunderstood how many accessions need adding. Is there any value in adding the  project and sample accessions (which was my assumption of what needed updating, sorry) 
-S-SUBS3 : BioStudies - linked to all BioSamples accessions
- PRJEB31843: ENA Study - Contains all data
and waiting for the spreadsheet updates to do the rest? I would have thought most people would have used these top level accessions? (but maybe that is also an incorrect assumption?)

**Comment by mshadbolt** [comment]
Yeah the project accessions are already on the browser  https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7

That's why I assumed you wanted me to edit sample level stuff. The ones that are missing are biosample accessions for some donors, specimens and cell suspensions. As well as run and experiment accessions for most of the sequence files (about 325 files)

I agree the most important are the project level ones that are already there, and I think I may go insane editing all of those sample/file ones by hand in the ui...so am hoping this one can wait until we are able to update via a script that calls the API or via spreadsheet. Ingest should kind of be aware of the accessions since we used the archiver, so it would be an extension of that software to actually add the accessions back to the metadata.

**Comment by lauraclarke** [comment]
So I think that for a project, having the run level accessions is useful, especially if we know that a study contains multiple different technologies as it gives them a way to get just the files from an archive they need for the subset of a study they want to download.

I agree that adding them after the fact if there is a high burden doesn't really add anything in the DCP context as they aren't ever exposed there

**Comment by mshadbolt** [comment]
I agree it is useful to have as many accessions available as possible in the metadata. I don't think it is ever something a wrangler should have to do manually though.

**Comment by Wkt8** [comment]
Going to talk to jacob about updating the biomaterial and sequence file accessions for a DCP1 project.

**Comment by Wkt8** [comment]
[https://app.zenhub.com/files/261790554/41d54993-e7d3-493b-818a-9fccb482b26d/download](https://app.zenhub.com/files/261790554/41d54993-e7d3-493b-818a-9fccb482b26d/download)
This is a spreadsheet that could go in to 'update' the DCP1 project. The updated sample accessions are highlighted in peach.

**Comment by Wkt8** [comment]
Tasks: 

- [ ] Update the project in the current submission via the spreadsheet tab (with the new spreadsheet with sample accessions)
- [ ] Check if the project passes graph validation
- [ ] Export the project (inform Jacob)
- [ ] Jacob should delete the relevant files from the staging area 
- [ ] Fill in the project export form

**Comment by aaclan-ebi** [comment]
@Wkt8 to look into graph validation errors re supplementary files.

**Comment by Wkt8** [comment]
This is throwing a graph-validation-error because:
- There is a supplementary file which is uploaded as CGaP_Human_Lung_Dissociation.docx
- There is a dissociation protocol which lists a file as the protocol_core.document, but it lists a pdf version "CGaP Human Lung Dissociation.pdf"
- Our metadata-schema only accepts protocol_core.documents as 'pdfs'
- There is an ingest-graph-validator test which requires files listed in protocol_core.document to also be listed in the supplementary_file tab, and cross-references filename to ensure they are identical.

As a result - the ingest-graph-validator test is failing, and i am unable to change the protocol_core.document to the appropriate file format of 'pdf'.

Going to ask jacob to set to valid and export, as we have done so for the other dcp1 projects.

**Comment by Wkt8** [comment]
Spoke to Jacob who set the graph state to valid, exported metadata only. Project is exporting. 
Tasks: 
- [ ] Confirm project has fully exported
- [ ] Inform Jacob to delete the relevant files from the staging area 
- [ ] Fill in project export form

**Comment by Wkt8** [comment]
Jacob has deleted files. Project exported with export form filled. Moving to finished.

**Comment by ESapenaVentura** [comment]
The run and sample accessions are not exposed anywhere in the DCP. There is virtually no way to verify that the updates have been applied
